back/graphql/type/users.js

```diff
@@ -0,0 +1,79 @@
+import { conn } from "../../db/connection.js";
+import {
+  GraphQLID,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql";
+
+export const getUserByUid = async (uid) => {
+  let result;
+  if (uid === undefined) {
+    result = await conn.execute(`SELECT *
+                                     FROM users`);
```

Comment:
Why you want to split the SQL statement into two lines?

Reply:
A kind of formatting style.